concent_api/core/tests/utils.py

```diff
@@ -1036,3 +1035,19 @@ def _get_verification_deadline_as_timestamp(
     def _get_uuid(last_char: Optional[str] = None) -> str:
         assert last_char is None or (isinstance(last_char, str) and len(last_char) == 1)
         return generate_uuid(last_char)
+
+    def send_request(
+        self,
+        url: str,
+        data: bytes,
```

Comment:
I think that sometimes data might not be available (ie. sending request to `protocol_constants`.).

Reply:
changed to `data: Optional[bytes] = None`